ODF-community.french.md

```diff
@@ -0,0 +1,526 @@
+Cadre de Décicion Ouverte
+=========================================
+
+Version Communautaire 1.0.3
+
+Dernière mise à jour de la version originale : 13 Oct. 2016
+
+Dernière mise à jour de la traduction : 21 Mars 2017 
+
+© 2014-2016 Red Hat et contributeurs | Le Cadre de Décision Ouverte ("Open
```

Comment:
This one is tricky : the original version says `© 2014-2016` so there's a question wether the translated copyright notice applies to the original version or the translated version. If I translate a copyright notice from 1992, I think I'm not supposed to modify the date...

Maybe we should have two copyright notice : one for the original content and one for the translation... 

I'm leaving it as-is for now...